api/graphql/mutations/role.js

```diff
@@ -67,20 +61,25 @@ export async function addRoleToMember ({userId, groupRoleId, personId, groupId})
   }
 }
 
-export async function removeRoleFromMember ({userId, memberRoleId, personId, groupId}){
+export async function removeRoleFromMember ({userId, groupRoleId, personId, groupId}){
   if (!userId) throw new GraphQLYogaError(`No userId passed into function`)
 
-  if (personId && memberRoleId && groupId) {
+  if (personId && groupRoleId && groupId) {
     const groupMembership = await GroupMembership.forIds(userId, groupId).fetch()
 
     if (groupMembership && (groupMembership.get('role') === GroupMembership.Role.MODERATOR) || userId === personId) {
```

Comment:
userId and personId?

Reply:
so this is prepping for a time when non moderators might be able to give people badges? i dont know if we really know if/what that use case might be, so not sure we need this. but open to it

Reply:
I just added the option for any user to be able to remove a badge from themselves in the API but there isn't that option int he front-end for now.